cdn/font-awesome/4.3.0/css/font-awesome.css

```diff
@@ -913,7 +913,6 @@
 .fa-tablet:before {
   content: "\f10a";
 }
-.fa-mobile-phone:before,
```

Comment:
Then also remove it from here: https://github.com/3scale/porta/blob/master/cdn/font-awesome/4.3.0/css/font-awesome.min.css

Reply:
We shouldn't even touch this files, my bad!

Reply:
@josemigallas plz remember to remove this change then